tests/fuzz/LD_PRELOAD/global_overrides.c

```diff
@@ -48,10 +48,11 @@ int s2n_stuffer_send_to_fd(struct s2n_stuffer *stuffer, int wfd, uint32_t len)
     }
 
     /* Otherwise, call the original s2n_stuffer_send_to_fd() */
-    typedef int (*orig_s2n_stuffer_send_to_fd_func_type)(struct s2n_stuffer *stuffer, int wfd, uint32_t len);
+    typedef int (*orig_s2n_stuffer_send_to_fd_func_type)(struct s2n_stuffer *stuffer, int wfd, uint32_t len, uint32_t *bytes_sent);
     orig_s2n_stuffer_send_to_fd_func_type orig_s2n_stuffer_send_to_fd;
     orig_s2n_stuffer_send_to_fd = (orig_s2n_stuffer_send_to_fd_func_type) dlsym(RTLD_NEXT, "s2n_stuffer_send_to_fd");
-    return orig_s2n_stuffer_send_to_fd(stuffer, wfd, len);
+    orig_s2n_stuffer_send_to_fd(stuffer, wfd, len, bytes_sent);
+    return *bytes_sent;
```

Comment:
Return success, right? Also, that call needs a GUARD

Reply:
Added it.